### DISCUSSION TITLE
Stylix compatibility

### DISCUSSION THREAD

**Discussion by noahfraiture** [Jas-SinghFSU/HyprPanel#334] [Q&A]
Hello,
How does it work with stylix ? It seems that it start with a default that matugen can modify. Is this undefined behavior or do we know what will happen ?

**Comment by anotherhadi** [comment]
> Hello,
> 
> How does it work with stylix ? It seems that it start with a default that matugen can modify. Is this undefined behavior or do we know what will happen ?

In my configuration, I'm just using the stylix variables and put them in my hyprpanel conf (https://github.com/anotherhadi/nixy/blob/main/home/system/hyprpanel/default.nix)

**Reply by noahfraiture** [reply]
Oh that's smart, thank you !

**Reply by noahfraiture** [reply]
I actually have an `issue`, hyprpanel use more colors theme instead of 16 bits color scheme like stylix does. How do you solve this issue ?

**Reply by anotherhadi** [reply]
> I actually have an `issue`, hyprpanel use more colors theme instead of 16 bits color scheme like stylix does. How do you solve this issue ?

I'm using less colors than the default theme. I mainly use 4 colors: background, background alt, accent & foreground.
I set (using the configuration file) the options to one of those 4 colors and that's it

**Reply by noahfraiture** [reply]
I see, I'll try that and see if it create some problem. Thank you (you have really nice config, I take some inspiration of it)
